config/settings/base.py

```diff
@@ -54,7 +54,6 @@
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
-    "django.contrib.auth.middleware.SessionAuthenticationMiddleware",
```

Comment:
This looks like it was just removed and has no functionality after 1.10. Seems fine.